aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -205,6 +206,30 @@ Tensor imag(const Tensor& self) {
 Tensor& conj_out(Tensor& result, const Tensor& self) { return unary_op_impl_out(result, self, conj_stub); }
 Tensor conj(const Tensor& self) { return unary_op_impl(self, at::conj_out); }
 
+Tensor conj_view(const Tensor& self) {
+ Tensor self_;
+  if (self.is_quantized()) {
+    auto impl = c10::make_intrusive<QTensorImpl>(
```

Comment:
I pulled the quantized logic from `alias_with_sizes_and_strides`, but I'm not actually sure if I need it.

Reply:
We don't have quantized complex, so probably not :)